### PR TITLE
Enable Copyright linter tests + Add Contributing.md file

### PR DESCRIPTION
**Pull request by Blast545**
Including the ament_lint_common skips the copyright test because the package does not contain source files. This PR adds explicitly the packages used so the copyright check is actually used (for LICENSE and Contributing.md files).

### CODE REVIEW

**Comment by Blast545** [comment]
Reference to CI linter tests skipping copyright check before this PR. 

[Build Status](http://ci.ros2.org/job/ci_linux/10674/)

**Comment by Blast545** [comment]
Copyright tests results for linux: [here](https://ci.ros2.org/job/ci_linux/10677/testReport/spdlog_vendor/copyright/)

Complete CI results:
* Linux [Build Status](http://ci.ros2.org/job/ci_linux/10677/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/6095/)
* macOS [Build Status](http://ci.ros2.org/job/ci_osx/8682/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/10575/)

**Review by brawner — COMMENTED** [review]
Same comment about the copyright test. I don't think it needs to run since there are no source files.

File: CMakeLists.txt [hunk 1]
```diff
@@ -83,9 +83,20 @@ if(NOT spdlog_FOUND OR "${spdlog_VERSION}" VERSION_LESS 1.5.0)
   build_spdlog()
 endif()
 
+# TODO(blast545): This section and package.xml should be updated once copyright auto is solved. See: https://github.com/ament/ament_lint/issues/237
```
Comment (brawner) [inline CMakeLists.txt:86]:
```suggestion
# TODO(blast545): This section and package.xml should be updated once copyright auto is solved.
# See: https://github.com/ament/ament_lint/issues/237
```

Keep ament_lint_auto as well?

**Review by brawner — APPROVED** [review]

**Pull request merged**